planb-cassandra.sh

```diff
@@ -65,11 +65,15 @@ echo $KEYSTORE | base64 -d > /etc/cassandra/keystore
 echo "Finished bootstrapping node."
 # Add route 53record seed1.${CLUSTER_NAME}.domain.tld ?
 
+ncores=$(grep -c ^processor /proc/cpuinfo)
 #
 # Assuming we are using SSD storage, set memtable_flush_writers to the
 # number of CPU cores:
 #
-export MEMTABLE_FLUSH_WRITERS=$(grep -c ^processor /proc/cpuinfo)
+export MEMTABLE_FLUSH_WRITERS=$ncores
+
+# the same for concurrent_compactors setting:
+export CONCURRENT_COMPACTORS=$ncores
```

Comment:
I advise not touching these defaults. These were exactly the settings that we tried to out in the past that took some nodes down